test/e2e/testnet/manifest.go

```diff
@@ -0,0 +1,77 @@
+package testnet
+
+import (
+	"time"
+
+	"github.com/celestiaorg/celestia-app/v2/app"
+	"github.com/celestiaorg/celestia-app/v2/app/encoding"
+	"github.com/celestiaorg/celestia-app/v2/test/util/genesis"
+	blobtypes "github.com/celestiaorg/celestia-app/v2/x/blob/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+)
+
+// Manifest defines the parameters for a testnet.
+type Manifest struct {
+	ChainID      string
+	TestDuration time.Duration
+	// Number of validators in the testnet
+	Validators int
+	TxClients  int
+	// Self-delegation amount for validators
+	SelfDelegation int64
+	// CelestiaAppVersion a specific version of the celestia-app container image within celestiaorg repository on GitHub's Container Registry i.e., https://github.com/celestiaorg/celestia-app/pkgs/container/celestia-app
+	CelestiaAppVersion string
+	// TxClientVersion  a specific version of the txsim container image within celestiaorg repository on GitHub's Container Registry, i.e., https://github.com/celestiaorg/celestia-app/pkgs/container/txsim
+	TxClientVersion string
```

Comment:
[not blocking] tx client or tx sim? Should we standardize on one term for it?

Reply:
I agree with this, went over the code and tried to replace all txsim usages with tx clients. Please see https://github.com/celestiaorg/celestia-app/commit/5eea75475272900b57d9a1cba565c65953d0f4ee.